scijava/scijava-ops-api/templates/main/java/org/scijava/ops/api/OpBuilder.vm

```diff
@@ -433,28 +433,38 @@ public class OpBuilder {
 			this.outType = outType;
 		}
 
+		/**
+		 * Match a {@link org.scijava.function.Functions} op to produce an instance of this builder's {@code outType}.
+		 *
+		 * @see #computer
+		 */
 		public $functionArity.call($arity)$generics.call($arity) function() {
 			return matchFunction(env, opName, $inputTypesWithOutput.call($arity));
 		}
 
+		/**
+		 * As {@link #function}, but match using the provided {@code Hints}.
+		 */
 		public $functionArity.call($arity)$generics.call($arity) function(final Hints hints) {
 			return matchFunction(env, opName, $inputTypesWithOutput.call($arity), hints);
 		}
 
+		/**
+		 * Match a {@link org.scijava.function.Computers} op to work with pre-allocated output.
+		 *
+		 * @see #function
```

Comment:
Same here: would be good to explain that the link to `#function` is by contrast/alternative, a different style of invocation than this one or some such. Also, why not also `@see #inplace` in that case?

Reply:
> Also, why not also `@see #inplace` in that case?

Note that this is in the `IT_OT` builders which do not have `inplace` methods. @gselzer @ctrueden is this a mistake? It seemed intentional to me that only the `IV_*` builders had `inplaces` but if the `IT_*`'s should too I'm happy to add them.